packages/xrpl/test/webpack.config.js

```diff
@@ -100,15 +100,11 @@ function webpackForTest(testFileName) {
       extensions: ['.ts', '.js', '.json'],
       fallback: {
         module: false,
-        assert: require.resolve('assert-browserify'),
```

Comment:
I think assert should still be required.  Maybe we dont have a test that hits any assets.

Reply:
There's another assert webpack config a couple of lines below - we don't need two